tools/mcp/daemon/sr_od_daemon.py

```
@@ -0,0 +1,230 @@
+    if existing_proc:
+        return f"[OK] 主 MCP server 已在运行 (PID: {existing_proc.pid})"
+
+    if is_port_in_use(port):
+        return f"[WARN] 端口 {port} 已被占用，可能有其他程序在使用"
+
+    try:
+        # 输出重定向到日志文件:长驻 server 若用 PIPE 且不持续消费,buffer 满会阻塞子进程
+        log_path = PROJECT_ROOT / '.debug' / 'sr_od_mcp' / 'main_server.log'
+        log_path.parent.mkdir(parents=True, exist_ok=True)
+        cmd = ['uv', 'run']
+        # .env 存在才追加 --env-file（对齐 GUI 的 mcp_service_interface._server_command，
```

Ruff (0.15.21)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)